eng/targets/Imports.targets

```diff
@@ -34,6 +34,18 @@
     <RoslynCheckCodeStyle Condition="'$(RoslynCheckCodeStyle)' == '' AND ('$(ContinuousIntegrationBuild)' != 'true' OR '$(RoslynEnforceCodeStyle)' == 'true')">true</RoslynCheckCodeStyle>
   </PropertyGroup>
 
+  <!--
+    PERF: Set default value for 'UseRoslynAnalyzers' to determine if analyzers should be executed.
+          Default to 'false' in all non-CI builds to improve local build time (i.e. csc/vbc invocations both inside Visual Studio and from command line prompt), except if:
+            1. We are enforcing code style, i.e. '$(RoslynEnforceCodeStyle)' == 'true' OR
+            2. We are explicitly running code analysis via "Run Code Analysis" command, i.e. '$(RunCodeAnalysis)' == 'true'.
+          Otherwise, default to 'true'.
+  -->
+  <PropertyGroup Condition="'$(UseRoslynAnalyzers)' == ''">
+    <UseRoslynAnalyzers Condition="'$(DesignTimeBuild)' != 'true' AND '$(ContinuousIntegrationBuild)' != 'true' AND !('$(RoslynEnforceCodeStyle)' == 'true' OR '$(RunCodeAnalysis)' == 'true')">false</UseRoslynAnalyzers>
```

Comment:
When does `$(RunCodeAnalysis)` get set to `true`? Not familiar with the workflows that enable that property.

Reply:
This property is set to true when you execute "Run Code Analysis" context command in Visual Studio, either from top level "Analyze" menu OR from solution explorer context menu for project/solution: https://docs.microsoft.com/en-us/visualstudio/code-quality/how-to-run-code-analysis-manually-for-managed-code?view=vs-2019. Starting VS2019 16.5, this command force completes all the applicable analyzers on the selected project/solution.